src/Modals/ModalNewForm/ModalNewForm.js

```diff
@@ -103,9 +104,17 @@ class PostForm extends Component {
 
                 <div className="postForm_question">
                     <div className="titles">
-                        In(city or zipcode): 
+                        In:  
+                        <select className={`citySelection ${this.getErrorMessage(this.state.address_city)} `} 
+                                onChange={(e)=>this.onChangeInput('address_city', e.target.value)}>
+                            <option value="">Select city</option>
```

Comment:
Can you create a "constants.js" filie and keep these in there? like `CITIES_OPTIONS = [...]` or something. Then this can just be a for loop and it'll be easy to add and remove cities